apps/sms/test/marionette/drafts_test.js

```diff
@@ -283,6 +289,34 @@ marionette('Messages Drafts', function() {
       });
     });
 
+    suite('Thread-less recipient-less drafts', function() {
```

Comment:
__nit:__ and please ```Conversation-less drafts``` :) I noticed that new contributors are confused by the fact that we use both ```conversation``` and ```thread``` across our codebase.